core/web/ocr_job_runs_controller.go

```diff
@@ -27,19 +27,36 @@ func (ocrjrc *OCRJobRunsController) Index(c *gin.Context) {
 	}
 
 	var pipelineRuns []pipeline.Run
+	var count int
+	err = ocrjrc.App.GetStore().DB.
```

Comment:
Could we encapsulate this part in the ORM and have a unit test for it?

Reply:
We've had a chat with @samsondav about not putting one-off queries into the ORM file. Also, I don't think there's any value in a unit test as we have controller tests cover this.

Reply:
I think you'd be better off having more satisfactory tests at an ORM like layer, rather than at a controller layer, way less setup and faster to run. Keeping this logic in the ORM would be more consistent with all of the other controllers we have, I'd prefer to keep patterns consistent rather than introduce new ones like this.

Reply:
Oh ORM is just one place, it could be in a service layer, or something like that, but just decoupled from setting up a whole HTTP router and all that stuff. This is making our entire test suite huge/slow :/

Reply:
What do you mean by "more comprehensive tests"? I only see a single path for this query (+ error, but I doubt we need to test cases where a column is missing 🤷‍♂️ ). Also, we definitely want controller tests (all too easy to have unit tests for each component and then a missing route 😅) so unit tests for this query would essentially be duplicated. _I think_.

Reply:
Yeah I'd just want to have a happy path controller test, then a series of very comprehensive tests for the "model" layer, for the query. e.g: no records, 1 record, `jobs.offchainreporting_oracle_spec_id IS NOT NULL` NULL vs not null to make sure inclusion, query with fewer results than page, more tests than page, that kind of thing.